pkgs/applications/editors/vim/plugins/non-generated/rainbow-delimiters-nvim/default.nix

```diff
@@ -1,18 +1,21 @@
 {
   lib,
   vimUtils,
-  fetchFromGitLab,
+  fetchFromGitHub,
   nix-update-script,
 }:
+let
+  version = "0.8.0";
+in
 vimUtils.buildVimPlugin {
   pname = "rainbow-delimiters.nvim";
-  version = "unstable-2025-01-12";
+  inherit version;
 
-  src = fetchFromGitLab {
+  src = fetchFromGitHub {
```

Comment:
If it can use GitHub, it should go to the automatic updater

Reply:
I went back and forth in my head, I don't mind it being here to get stable updates until we figure out the stable pinning inside the main set. Not sure what everyone else thinks if we want to just roll back there again for unstable updates again

Reply:
Oh wait, GItHub is just a mirror

https://github.com/HiPhish/rainbow-delimiters.nvim/pull/105#issuecomment-2016751535

We should not use mirrors without a reason, and instead target the main developer platform (IMO)